src/screens/ENSConfirmRegisterSheet.js

```diff
@@ -94,7 +96,14 @@ export default function ENSConfirmRegisterSheet() {
     mode,
   } = useENSRegistration();
 
-  const accentColor = useRecoilValue(accentColorAtom);
+  const [accentColor, setAccentColor] = useRecoilState(accentColorAtom);
+
+  const { result: dominantColor } = usePersistentDominantColorFromImage(
+    initialAvatarUrl || ''
+  );
+  useEffect(() => {
+    setAccentColor(dominantColor || colors.purple);
+  }, [dominantColor, setAccentColor]);
```

Comment:
Is this needed? Should automatically set the dominant color in the other component